src/otg/dataset/intervals.py

```diff
@@ -72,8 +72,8 @@ def from_source(
             raise ValueError(f"Unknown interval source: {source_name}")
 
         source_class = source_to_class[source_name]
-        data = source_class.read(spark, source_path)
-        return source_class.parse(data, gene_index, lift)
+        data = source_class.read(spark, source_path)  # type: ignore
```

Comment:
Do these comments `# type: ignore` have any meaning?

Reply:
it's telling mypy not to handle this line. This code would need a proper refactor but I thought it was not worth the effort.